### DISCUSSION TITLE
Need Some Help

### DISCUSSION THREAD

**Discussion by jltnol** [ACDR/homebridge-yamaha-avr#57] [General]
So I am admittedly way out of my league here, but do have a question, which I hope will be simple to answer.  I've managed to get Homebridge installed on my computer, and have a plugin for my LG WebOS up and running.  I've downloaded your plugin, and have also been successful in getting it installed and configured.  I can control my receiver from HomeKit as expected.

What I'd LIKE to do is to set up an automation In HomeKit where when a certain input on my Yamaha Receiver is selected, the TV turns on.  Just looking at it, that doesn't seem possible, because in HomeKit when I get to the "Accessory Automation" page, there really isn't an option for this. 

Just want to make sure that I am right in my thinking, and that I'm just not missing a simple step to accomplish what I'm trying to automate....

**Comment by Kaici433** [comment]
I think you are right and it is not possible in HomeKit directly.
